### ISSUE TITLE
Error loading syntax file

### ISSUE THREAD

**Issue by cmatzka** [coronalabs/CoronaSDK-SublimeText#37]
v1.7.6, gives me the error

Error loading syntax file "Packages/CoronaSDK-SublimeText/CoronaSDKLua.sublime-syntax": Unable to read Packages/CoronaSDK-SublimeText/CoronaSDKLua.sublime-syntax

As a result, syntax highlighting doesn't work.
Using latest Sublime v3.2.2, build 3211 on Windows.

**Comment by sekodev** [comment]
Can you do a clean install of the editor to see if it works?
https://github.com/coronalabs/CoronaSDK-SublimeText#installation-instructions

**Comment by cmatzka** [comment]
I uninstalled and reinstalled before reporting the issue but I'll try that again later today.

**Comment by cmatzka** [comment]
Tested again. Uninstall didn't work and reported the same issue. I then uninstalled and reinstalled Sublime, installed the package and now everything works. Seems like the update didn't work for me. Anyways, glad it's fine now. Thanks!

**Comment by cmatzka** [comment]
Unfortunately after trying to save a file it gave me the same error again... so, the whole reinstallation didn't solve the issue.

**Comment by sekodev** [comment]
That's pretty interesting. Clean install should have solved the issue. Are you installing the package or referencing the repository? Do you have any custom settings for the editor or Sublime Text?

**Comment by cmatzka** [comment]
I installed through Package Control: Install Package. I'm not using any custom settings. I did a clean install of Sublime and right after that installed the package.

**Comment by sekodev** [comment]
Looking back at your first post, "unable to read" error made me think that there might be a permission issue here because that file is actually included in the package. You can see it in the Code section.

If it's a permission issue, you should look into that but if it's not, I have one last suggestion. Following the same installation instructions above, can you remove the package and switch to repository instead?

**Comment by mmaslati** [comment]
I'm having the same issue today. All of a started today with no changes I've made to Sublime at all. I started writing with a different MacOS User and noticed the issue, switched to default user which I worked with yesterday with no issues, and the issue happens here as well.
When opening a Lua file the syntax shows 'Corona Lua', and the coloring is OK. As soon as I save a change text turn white, under View=>Syntax, nothing is selected, and the bottom right corner changes from "Corona Lua" to "CoronaSDKLua" (which is not an option under the syntax list).

Tried uninstalling and reinstalling the package using Package Control and the issue persists.
How do I connect through Git to Sublime Text?
